### ISSUE TITLE
Question about loss

### ISSUE THREAD

**Issue by e2sh83** [smallcorgi/3D-Deepbox#9]
Hi @smallcorgi 

What loss value did you arrive after 50 epoch training?
I got about 16.0 which seems higher than my expectation(starting from 19.3). The orientation value regressed during test seems reasonable. I am just wondering your case.
Thank you for your answer in advance.

**Comment by smallcorgi** [comment]
Hi @e2sh83 ,

Sorry, I didn't log the training loss.

But, I had trained the Deep3DBox only 20 epochs and evaluated the AP of orientation.

The following table is the result.

| Easy       | Moderate     |  Hard |
|-------------|--------|-------- |
| Car   |    98.75% | 96.42%  | 92.92% |
|Pedestrian|  65.12% | 64.11%  | 63.98% |
|Cyclist   | 77.5%  | 75.46% | 72.95%|

**Comment by hectorgui** [comment]
@e2sh83 same as your result 
@smallcorgi it seems that your tesing data is your training data?

**Comment by smallcorgi** [comment]
Hi @hectorgui,
I split the KITTI  into train/val set used in 3DOP/Mono3D/MV3D.
And my result is testing on the validation set.
